### PR TITLE
conflicts introduction

### PR DESCRIPTION
**Pull request by mikanchu**

### CODE REVIEW

File: Github - HackNight/README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ Pull Request, Flow Control, Merge conflicts, commits, stashes, repositories, ...
   `clone, merge, push, pull fetch`
 - Git branches
 - Git Rebase
+  - `git rebase` vs `git merge`
```
Comment (mikanchu) [inline Github - HackNight/README.md]: This is a proposal to take into account only if there's time

Comment (bertadevant) [reply Github - HackNight/README.md]: Could you write it just in case? I would prefer to have it

File: Github - HackNight/README.md [hunk 2]
```diff
@@ -35,4 +36,4 @@ If you have any questions or need any help you can reach out to:
 Women Who Code Barcelona: [barcelona@womenwhocode.com](mailto:barcelona@womenwhocode.com)
 
 Berta Devant: [@bertadevant](https://github.com/bertadevant)
-Helena He: [@mikanchu](https://twitter.com/mikanchu)
+Elena He: [@mikanchu](https://twitter.com/mikanchu)
```
Comment (bertadevant) [inline Github - HackNight/README.md]: sorry 😄

File: Github - HackNight/git_conflicts.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+# Instructions to create conflicts
+
+There are several types of conflicts. Merge conflics, rebase conflicts...
+It happens because git is confused and needs help to make the right decision
+
+## How to avoid conflicts
+Some recomendations to avoid conflicts:
+  - Use branches
+  - Sync from remote often (with `git fetch <remote>`)
+  - Rebase before pushing anything (with `git rebase branch`)
```
Comment (bertadevant) [inline Github - HackNight/git_conflicts.md]: Can we add a link to the documentation for rebases?

File: Github - HackNight/README.md (hunk 2 above)
Comment (mikanchu) [reply Github - HackNight/README.md]: 🤣

File: Github - HackNight/README.md (hunk 1 above)
Comment (mikanchu) [reply Github - HackNight/README.md]: I've created an issue #11

File: Github - HackNight/git_conflicts.md (hunk 3 above)
Comment (mikanchu) [reply Github - HackNight/git_conflicts.md]: added on #10

**Pull request merged**